### PR TITLE
Adds a 'request-version-update' label to all updater PRs and issues

### PR DESCRIPTION
**Pull request by patflynn**

### CODE REVIEW

**Comment by patflynn** [comment]
Tested e2e using the CLI manually.

File: pkg/gh/issues.go [hunk 1]
```diff
@@ -114,6 +116,17 @@ func (o GitOptions) CommentIssue(ctx context.Context, owner, repo, comment strin
 	return issue.GetHTMLURL(), err
 }
 
+func (o GitOptions) LabelIssue(ctx context.Context, owner, repo string, number int, labels *[]string) error {
```
Comment (luhring) [inline pkg/gh/issues.go:119]: Is there any sense in testing that the labeling is happening when we expect?

**Review by luhring — DISMISSED** [review]

File: pkg/update/update.go [hunk 2]
```diff
@@ -66,6 +66,7 @@ const (
   <img src="https://raw.githubusercontent.com/wolfi-dev/.github/b535a42419ce0edb3c144c0edcff55a62b8ec1f8/profile/wolfi-logo-light-mode.svg" />
 </p>
 `
+	updateLabel = "request-version-update"
```
Comment (kaniini) [inline pkg/update/update.go:69]: Does it make sense to allow the updateLabel to be customized?

File: pkg/gh/issues.go (hunk 1 above)
Comment (patflynn) [reply pkg/gh/issues.go:119]: Are you thinking verifying that the call succeeded here by looking at the returned issue and verifying its state against the arguments, or an end-to-end test to github in the test suite, or both? :)

Comment (luhring) [reply pkg/gh/issues.go]: Yeah, good point, maybe an E2E test makes more sense. And maybe not now if we don't have a precedent for it. Treat this as optional — LGTM!

File: pkg/update/update.go (hunk 2 above)
Comment (patflynn) [reply pkg/update/update.go]: https://github.com/wolfi-dev/os/pull/1938

Comment (patflynn) [reply pkg/update/update.go]: Updated the code to pass labels as a flag and tested manually: https://github.com/wolfi-dev/os/pull/1938

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**